ui/src/support/types.ts

```diff
@@ -0,0 +1,52 @@
+export enum MENU_BUTTONS_WITHOUT_DROP_DOWN {
+    HOME = "Home",
+    WHY_GITLAB = "Why GitLab",
+    PLATFORM = "Platform",
+    PRICING = "Pricing"
+}
+
+
+export enum MENU_BUTTONS_WITH_DROP_DOWN {
+    SOLUTIONS = "Solutions",
+    RESOURCES = "Resources",
+    COMPANY = "Company",
+    CONTACT = "Contact us"
+}
+
+export enum COMPANY_DROP_DOWN {
+    ABOUT = "About ",
+    JOBS = "Jobs ",
+    EVENTS = "Events ",
+    LEADERSHIP = "Leadership ",
+    TEAM = "Team ",
+    HANDBOOK = "Handbook ",
+    INVESTOR = "Investor relations ",
+    TRUST = "Trust Center ",
+    NEWSLETTER = "Newsletter ",
+    PRESS = "Press "
+}
+
+export enum CONTACT_US_DROP_DOWN {
```

Comment:
As far as I can see these enum properties are not called anywhere in your code. Please delete it.

Reply:
done